src/mame/force/miniforce.cpp

```diff
@@ -105,21 +105,17 @@
 #include "bus/vme/vme_hcpu30.h"
 #include "machine/clock.h"
 
-#define LOG_GENERAL 0x01
-#define LOG_SETUP   0x02
-#define LOG_PRINTF  0x04
+#define LOG_SETUP   (1U << 1)
+#define LOG_PRINTF  (0)
 
-#define VERBOSE 0 // (LOG_PRINTF | LOG_SETUP  | LOG_GENERAL)
+#define VERBOSE (0) // (LOG_SETUP | LOG_GENERAL)
 
-#define LOGMASK(mask, ...)   do { if (VERBOSE & mask) logerror(__VA_ARGS__); } while (0)
-#define LOGLEVEL(mask, level, ...) do { if ((VERBOSE & mask) >= level) logerror(__VA_ARGS__); } while (0)
+#if VERBOSE && LOG_PRINTF
+#define LOG_OUTPUT_FUNC printf
+#endif
 
-#define LOG(...)      LOGMASK(LOG_GENERAL, __VA_ARGS__)
-#define LOGSETUP(...) LOGMASK(LOG_SETUP,   __VA_ARGS__)
+#include "logmacro.h"
 
-#if VERBOSE & LOG_PRINTF
-#define logerror printf
-#endif
```

Comment:
This is another one with the `LOG_PRINTF` that should go away.

Reply:
Addressed in the same way was previous VME files.